### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by oovillagran**
<h1>Hi, there 👋</h1>

In this project, I created a full website for the Math magicians app, consisting of three pages and using the components created before.

**Project requirements**
- Create a website consisting of 3 pages: Home, Calculator, and Quote.
- Add your own styles to improve the look and feel.
- For every page, you should create a <Route>.
- Make sure that navigation links for all 3 routes are displayed on each page.
- Use [React Router](https://reactrouter.com/) to make the routes work.
- The general layout should match these wireframes:

<h2>Thanks for your kind review 🙌</h2>

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
+  );
+  // <>
+}
+//   <div className="App">
+//     <h1 className="app-title">Math Magicians</h1>
+//   </div>
+//   <Calculator />
+//   <Quote />
+// </>
+// );
```
Comment (DuaneDave) [inline src/App.js]: In order to enforce best practices in your code base, consider not leaving behind commented pieces of code.

File: src/App.js [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
+  return (
+    <>
+      <Navigation />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/calculator" element={<Calculator />} />
+        <Route path="/quote" element={<Quote />} />
+      </Routes>
+    </>
+  );
```
Comment (DuaneDave) [inline src/App.js:18]: [OPTIONAL] When I load your app on the browser, these error pops up. Consider resolving them in order to follow best practices for indentation format.

err

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @oovillagran  👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ The styling of your app is awesome
✔️ Very intuitive of you to have a loading state on the quote route


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by oovillagran** [comment]
Thanks @DuaneDave

File: src/App.js (hunk 1 above)
Comment (Abenezer-Tilahun) [reply src/App.js]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @oovillagran  
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git -Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.


## Optional suggestions 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by oovillagran** [comment]
Thanks @Abenezer-Tilahun !

**Pull request merged**